mkdocs.yml

```diff
@@ -145,6 +145,7 @@ nav:
             - Multi-file upload: administration/back_office/multifile_upload.md
             - Sub-items list: administration/back_office/subitems_list.md
             - Notifications: administration/back_office/notifications.md
+            - Customize search sorting: administration/back_office/search_sorting.md
```

Comment:
```suggestion
            - Customize search sorting: administration/back_office/customize_search_sorting.md
```

Reply:
Done in 5527c36956be0e2296ad36fbdfb8b6386afec4dc